### DISCUSSION TITLE
how to add vue-router redirect

### DISCUSSION THREAD

**Discussion by cipami** [nuxt/nuxt#16667] [Questions] [answered]
Is there a way to extend routes in general? Or in my case to add redirect without creating a middleware?
https://router.vuejs.org/guide/essentials/redirect-and-alias.html#redirect

**Comment by TheAlexLichter** [comment, marked as answer]
You can use Nuxt hooks [(`pages:extend`)](https://github.com/nuxt/framework/blob/main/packages/schema/src/types/hooks.ts#L78) or e.g. a `router.options.ts` - https://github.com/nuxt/framework/pull/3485
